src/Admin/AbstractAdmin.php

```diff
@@ -3571,6 +3571,8 @@ protected function getSubClass($name)
 
     /**
      * Attach the inline validator to the model metadata, this must be done once per admin.
+     *
+     * @deprecated since sonata-admin/admin-bundle 3.81
      */
     protected function attachInlineValidator()
```

Comment:
When we deprecate a method, we generally add a 
```
@trigger_error(sprintf('Some message'), E_USER_DEPRECATED);
```

Since here we're still using the method, we have one trick.
When we're calling the method, we're doing
```
$this->attachInlineValidator('sonata_deprecation_mute');
```
And in the method

```
if ('sonata_deprecation_mute' !== \fct_get_args[0] ?? null) {
    @trigger_error(...)
}
```

There are multiple example in the code.